static/js/unread_ui.js

```diff
@@ -82,7 +82,7 @@ function consider_bankruptcy() {
     }
 
     var now = new XDate(true).getTime() / 1000;
-    if ((page_params.unread_count > 500) &&
+    if ((page_params.unread_msgs.count > 500) &&
```

Comment:
I just didn't get around to it, and that was going to be a separate commit, but it also might make sense to squash them, assuming we're 100% sure the mobile app didn't use unread_count.